test/constants.js

```diff
@@ -4,7 +4,8 @@ const constants = {
 		RAF_THRESHOLD : 16,
 		SMALL: 5
 	},
-	ITEM_TO_OBSERVE: 5
+	ITEM_TO_OBSERVE: 5,
+	SKIP_CYCLE: 3
```

Comment:
We basically skip 3 cycles and 4th one we enter the loop. I understand the confusion here. I will update those two be consistent.